migrations/Version20230721121011.php

```diff
@@ -20,17 +20,21 @@ public function getDescription(): string
     public function up(Schema $schema): void
     {
         // this up() migration is auto-generated, please modify it to your needs
-        $this->addSql('CREATE TABLE AuthorizedDomain (
+        if ($schema->hasTable('synolia_authorized_domain')) {
+            return;
+        }
+
+        $this->addSql('CREATE TABLE synolia_authorized_domain (
           id INT AUTO_INCREMENT NOT NULL,
           name VARCHAR(255) NOT NULL,
-          isEnabled TINYINT(1) NOT NULL,
+          is_enabled TINYINT(1) NOT NULL,
```

Comment:
ℹ️ Not a BC because of, this change is also done in the same tag (1.0.2) and table name too.